actions/os_deferred_event_actions.py

```diff
@@ -86,6 +84,17 @@ def run_deferred_hooks(args):
                 ','.join(ovsdb.db_sb_connection_strs),
                 reactive.is_flag_set('config.changed.disable-mlockall'),
                 check_deferred_events=False)
+
+
+def run_deferred_hooks(args):
+    """Run deferred hooks.
+
+    :param args: Unused
+    :type args: List[str]
+    """
+    _run_deferred_hooks()
+    os_utils.restart_services_action(deferred_only=True)
```

Comment:
I lost a bit track of the big picture: is it because running deferred hooks may mark some services as requiring a restart? Is this going to restart only the ones that have been marked as needing one or is this going to restart all services that have considered deferable? I double-checked and the argument `deferred_only` is not documented and I got a bit lost in its implementation.

Reply:
Yes, running deferred hooks may mark some services as requiring a restart so this restarts them. The deferred_only kwarg indicates that only services that have been marked as needing a restart are going to be restarted